feed/models.py

```diff
@@ -29,7 +27,7 @@ class Link(models.Model):
     description = models.TextField()
     created_at = models.DateTimeField(auto_now_add=True)
     flag = models.IntegerField(default=0)
-    star = models.ManyToManyField(Member, blank=True, null=True, related_name="starred")
+    star = models.ManyToManyField(Member, blank=True, related_name="starred")
```

Comment:
This should be `submitted_by` = models.ManyToManyField(Member, related_name="links"` and we should move starred links to its own model